MemberExpectations.md

```diff
@@ -0,0 +1,63 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
+
+Members are encouraged to have individual opinions and are encouraged
+to express those opinions during discussions within the organization. While
```

Comment:
It's not so much that we are "encouraged" to have individual opinions, it's more that we recognize that each of us has individual opinions and will express those. It's a subtle but important difference. I do not need the projects permission or encouragement to bring my own point of view.